src/declarations/stencil-public-docs.ts

```diff
@@ -101,6 +123,7 @@ export interface JsonDocsMethod {
   signature: string;
   returns: JsonDocsMethodReturn;
   parameters: JsonDocMethodParameter[];
+  complexType: ComponentCompilerMethodComplexType;
```

Comment:
sure, I'll add a note about this and about the `JsonDocsEvent` - I didn't already because I thought it wasn't necessary for adding a new field, only for removing or changing an existing one